PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -390,6 +439,8 @@ log_Msg "AvgrdcSTRING: ${AvgrdcSTRING}"
 log_Msg "TopupConfig: ${TopupConfig}"
 log_Msg "BiasFieldSmoothingSigma: ${BiasFieldSmoothingSigma}"
 log_Msg "UseJacobian: ${UseJacobian}"
+log_Msg "T1wBiasCorrect: ${T1wBiasCorrect}"
+log_Msg "CustomBrain: ${CustomBrain}"
```

Comment:
Include reporting of MPPMode value as well.

Reply:
Added.